.drone.yml

```diff
@@ -278,15 +279,19 @@ steps:
   - name: acceptance_tests_uat
     <<: *acceptance_tests
     commands:
-      - export ACCEPTANCE_HOST_NAME=https://$${PROD_ENV}.uat.internal.sas-notprod.homeoffice.gov.uk
-      - export CUCUMBER_PUBLISH_ENABLED=true
+      # - export ACCEPTANCE_HOST_NAME=https://$${PROD_ENV}.uat.internal.sas-notprod.homeoffice.gov.uk
+      # - export CUCUMBER_PUBLISH_ENABLED=true
+      # - npx playwright install
+      # - npm run test:acceptance
+      - export NODE_ENV=development
+      - export ACCEPTANCE_HOST_NAME=https://dev.notprod.$${APP_NAME}.homeoffice.gov.uk
       - npx playwright install
       - npm run test:acceptance
     when:
       branch: master
       event: push
 
-
+  
```

Comment:
keep the blank line but remove the space at the beginning

Reply:
done